### DISCUSSION TITLE
¿Puedo unirme a un grupo?

### DISCUSSION THREAD

**Discussion by BrahianM** [rramosp/ai4eng.v1#151] [UDEA Students]
Buenos días profe y Felipe.
El día de ayer mis compañeros de equipo me abandonaron (eramos tres), decidieron cancelar la materia y ahora estoy solito, entoces quería preguntar si es posible que se me permita unirme a un grupo incompleto o si será obligatorio continuar con el desarrollo del proyecto yo solo. 
También tengo las capturas de pantalla como prueba de que se tiraron del barco justo ayer.
Quedo atento y gracias.

**Comment by Felipe-RA** [comment]
Saludos BrahianM

Puedes buscar otro grupo o trabajar sólo, como gustes!

**Comment by rramosp** [comment]
te puedes unir a un grupo incompleto, pero el total de integrantes no ha de
sobrepasar lo establecido en la página del curso

On Thu, Jun 30, 2022 at 7:41 AM BrahianM ***@***.***> wrote:

> Buenos días profe y Felipe.
> El día de ayer mis compañeros de equipo me abandonaron (eramos tres),
> decidieron cancelar la materia y ahora estoy solito, entoces quería
> preguntar si es posible que se me permita unirme a un grupo incompleto o si
> será obligatorio continuar con el desarrollo del proyecto yo solo.
> También tengo las capturas de pantalla como prueba de que se tiraron del
> barco justo ayer.
> Quedo atento y gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/151>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZILLXEPUL32RXRRXKZ3VRWIZBANCNFSM52JG77MQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
